frame/support/test/tests/benchmark_ui/bad_return_type_blank_with_question.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0277]: the `?` operator can only be used in a function that returns `Result` or `Option` (or another type that implements `FromResidual`)
+  --> tests/benchmark_ui/bad_return_type_blank_with_question.rs:15:14
+   |
+5  | #[benchmarks]
+   | ------------- this function should return `Result` or `Option` to accept `?`
```

Comment:
I'm a bit annoyed by the span of this error, but I don't think we can do anything here as it's generated by the compiler.

Reply:
yeah, this was actually slightly better than the custom one I tried to write in #13277 but I agree